### ISSUE TITLE
Editor: Worldmap doesn't have all of the blocks from the main game

### ISSUE THREAD

**Issue by OperationGT** [SuperTux/supertux#500]
I have discovered that the worldmap for the latest in-game editor does not have all of the original blocks in. This includes things such as mountains and lighter ice. Is this intended?

**Comment by Hume2** [comment]
There are two worldmap tilesets, where none of them contains all tiles. It needs something to switch tilesets.

**Comment by Hume2** [comment]
#502 should make you able to choose another tileset.

**Comment by OperationGT** [comment]
How do I choose another tileset, in simpler terms? Sorry, I don't really understand what was said in #502

**Comment by maxteufel** [comment]
@OperationGT currently, you can't in the in-game editor as #502 hasn't been merged yet. That also means it's not yet in the nightly builds. When it is merged, you'll be able to select one in a "Tile set" option in the level properties menu.

**Comment by OperationGT** [comment]
Ah ok, thanks

**Comment by maxteufel** [comment]
#502 is merged now and can be tested in the nightly or RC 2 builds. That means you can change the tileset in the editor.

However, forest and new ice tiles can't currently be used together, because the tilesets aren't merged. That issue will be tracked in SuperTux/data#27. That issue will be fixed after 0.5.0 though, because there is currently no need for that change in the official parts of the game.
